File: grind-169/1.two-sum/two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int, int> hashmap;
        for (int i = 0; i < nums.size(); i++) {
            int complement = target - nums[i];
            if (hashmap.find(complement) == hashmap.end()) {
                hashmap[nums[i]] = i;
            } else {
                return vector<int>{hashmap[complement], i};
            }
        }
        return {};
    }
};

File: grind-169/1.two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        dct = {}
        for index, value in enumerate(nums):
            find = target - value
            if find in dct:
                return [dct[find], index]
            dct[value] = index
